frontend/simple/views/CreateGroup.vue

```diff
@@ -236,22 +131,82 @@ export default {
       } catch (error) {
         console.error(error)
         this.errorMsg = L('Failed to Create Group')
+        return
+      }
+
+      try {
+        this.errorMsg = null
+        // TODO: as invitees are successfully invited display in a
+        // seperate invitees grid and add them to some validation for duplicate invites
+        for (let invitee of this.form.invitees) {
+          // We need to have the latest mailbox attribute for the user
+          const mailbox = await backend.latestHash(invitee.state.attributes.mailbox)
+          const sentDate = new Date().toString()
+
+          // We need to post the invite to the users' mailbox contract
+          const invite = new Events.HashableMailboxPostMessage(
+            {
+              from: this.$store.getters.currentGroupState.groupName,
+              headers: [this.$store.state.currentGroupId],
+              messageType: Events.HashableMailboxPostMessage.TypeInvite,
+              sentDate
+            },
+            mailbox
+          )
+          await backend.publishLogEntry(invitee.state.attributes.mailbox, invite)
+
+          // We need to make a record of the invitation in the group's contract
+          const latest = await backend.latestHash(this.$store.state.currentGroupId)
+          const invited = new Events.HashableGroupRecordInvitation(
+            {
+              username: invitee.state.attributes.name,
+              inviteHash: invite.toHash(),
+              sentDate
+            },
+            latest
+          )
+          await backend.publishLogEntry(this.$store.state.currentGroupId, invited)
+        }
+      } catch (error) {
+        console.error(error)
+        // TODO: Create More descriptive errors
+        this.errorMsg = L('Failed to Invite Users')
       }
     }
   },
   data () {
-    // TODO: wrap this in a 'form' parameter (#297)
     return {
-      groupName: null,
-      sharedValues: null,
-      changePercentage: 80,
-      memberApprovalPercentage: 80,
-      memberRemovalPercentage: 80,
-      incomeProvided: null,
-      contributionPrivacy: 'Very Private',
+      form: {
+        groupName: '',
+        sharedValues: null,
+        changePercentage: 80,
+        memberApprovalPercentage: 80,
+        memberRemovalPercentage: 80,
+        incomeProvided: null,
+        invitees: []
+      },
+      // todo: move these under appropriate key for #297
       errorMsg: null,
-      // this determines whether or not to render proxy components for nightmare
-      dev: process.env.NODE_ENV === 'development'
+      validity: {
+        groupName: false,
+        sharedValues: false,
+        incomeProvided: false
+      },
```

Comment:
What's going on with this duplication of fields?

I'm not sure the `validity` key is necessary. Remember we are already using the [vee-validate](https://github.com/baianat/vee-validate) validation plugin. Is it possible to use that with a combination of checking the values in `form` instead?

Reply:
If you did explore vee-validate and decided it wasn't the right way to go, I'd like to understand what you found out, i.e. the reasons why, and if they're good reasons, then they should be written up as comments right here above line 190.